docs/cse/rules/write-outlier-rule.md

```diff
@@ -0,0 +1,109 @@
+
+## Example rule
+The screenshot below shows an Outlier rule in the CSE rules editor. For an explanation of the configuration options, see [Configure an Outlier rule](#configure-an-outlier-rule), below. 
+
+<img src={useBaseUrl('img/cse/outlier-rule.png')} alt="Example Outlier Rule Definition"/>
+
+
+## Configure an Outlier rule
+This section has instructions for configuring an Outlier rule.
+
+### If triggered
+The settings in the **If triggered** section are divided into two subsections, one for providing Baseline configuration, and the other for Outlier model configuration.
+
+**Baseline Configuration**
+1.  **For the records matching the expression**. Enter an expression that matches the Records that you want to rule to apply to.
+1. **build a daily/hourly baseline**. Select the time window for building the baseline. It can either be a daily or hourly baseline.
+1. **for the entity(ies)**. Select one or more Record fields for which you want baselines built. Selecting multiple fields will build a distinct baseline for a combination of entities.
+1. Set the baseline and retention settings:
+   1. **Baseline Retention Period (days)**. The number of days after which the data points in the baseline will expire (be dropped from the baseline). The default is 90 days. You can decrease this period, but not increase it.
+   1. **Baseline Learning Period (days)**. The minimum amount of time for which data points should be collected before firing a Signal. The default is 30 days.
+   :::note
+   The **Baseline Learning Period** must be shorter than the **Baseline Retention Period**. Also be aware that short baseline learning periods may generate false positive Signals.
+   :::
+
+**Outlier Model Configuration**
+1. **Detect an outlier for**. Select the aggregate function that applies to the field in the matched Records to build a normal behavior baseline on.
+1. **of the Record field**. Select one or more Record fields to build a baseline on and detect an Outlier Signal.
+1. **Advanced Expression** (optional). When selected, disables the record field selector and allows defining Record fields within the field window.
+1. **Model Sensitivity Threshold** (1-5). Select the sensitivity of the model defined above. This is the number of standard deviations from the mean that the outlier model should consider for creating a Signal. Lower threshold corresponds to a more sensitive model resulting in more Signals.
+1. **Minimum Count Value** (default value 1). Enter the absolute minimum value below which an Outlier Signal will not be generated.
+
+### Then create a Signal
+
+For instructions, see [Configure “Then Create a Signal” settings](/docs/cse/rules/write-match-rule.md#configure-then-create-a-signal-settings) section of the Match Rule topic.
```

Comment:
```suggestion
For instructions, see [Configure “Then Create a Signal” settings](/docs/cse/rules/write-match-rule/#configure-then-create-a-signal-settings) section of the Match Rule topic.
```

Reply:
That .md got there as a copy-and-paste from the "Write a First Seen rule" article.